tests/sdk.test.ts

```diff
@@ -1,94 +1,100 @@
-
 import Web3 from "web3";
-import { ethers } from "ethers";
 import Assets from "../src/assets.json";
-import { ExternalProvider, Web3Provider } from "@ethersproject/providers";
-import { Synths } from "../src";
-import { Asset } from "../src/library/Asset";
+import {ExternalProvider, Web3Provider} from "@ethersproject/providers";
+import {Synths} from "../src";
+import {Asset} from "../src/library/Asset";
 
 let SDK: Synths;
 let UGAS0221: Asset;
-let globals: any;
 const network = "mainnet";
 const account = "0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be";
 
 beforeAll(async () => {
-  const web3 = await new Web3(new Web3.providers.HttpProvider(process.env.INFURA_URL_HTTP || ""));
-  const provider: ExternalProvider = (new Web3Provider(web3.currentProvider as any)).provider;
-  SDK = await new Synths({
-    provider: provider,
-    network: network,
-    account: account,
-  });
-
-  UGAS0221 = await new Asset(SDK.synthetics["ugas"]["0221"], SDK.options);
+    const web3 = await new Web3(new Web3.providers.HttpProvider(process.env.INFURA_URL_HTTP || ""));
+    const provider: ExternalProvider = (new Web3Provider(web3.currentProvider as ExternalProvider)).provider;
+    SDK = await new Synths({
+        provider: provider,
+        network: network,
+        account: account,
+    });
+
+    UGAS0221 = await new Asset(SDK.synthetics["ugas"]["0221"].asset, SDK.options);
 });
 
 describe('SDK Initialization', () => {
+    it("sdk network must be a string: \"mainnet\" or \"kovan\"", async () => {
+        expect(SDK.network).toBe("mainnet" || "kovan");
 
-  it("sdk network must be a string: \"mainnet\" or \"kovan\"", async () => {
-    expect(SDK.network).toBe("mainnet" || "kovan");
-  });
+    });
 
-  it("sdk must return the user account attached", async () => {
-    expect(SDK.account).toBe(account);
-  });
+    it("sdk must return the user account attached", async () => {
+        expect(SDK.account).toBe(account);
+    });
 
-  it("sdk must contain the assets.json assets", async () => {
-    expect(SDK.assets["ugas"][1].name).toBe(Assets[network]["ugas"][1].name);
-  });
+    it("sdk must contain the assets.json assets", async () => {
+        expect(SDK.assets.ugas[1].name).toBe(Assets[network]["ugas"][1].name);
```

Comment:
Typing is now accurate, but I think I accessed SDK.assets["ugas"] as SDK.assets.ugas here by accident. 

@byterose Do you think we should access through ["ugas"] or .ugas ? I'll make access uniform once I get your input.

Reply:
I updated all of the sdk tests, you can find them in ```./test/SynthsSdk.test```. Some methods and their tests are still missing but at least the existing once are now more accurate.